src/modules/organizations/containers/organization-container.jsx

```diff
@@ -22,13 +23,25 @@ class OrganizationContainer extends React.Component {
     this.props.dispatch(setCurrentOrganization(null));
   }
 
+  updateOrganization(organizationFields) {
+    organizationFields.name = organizationFields.display_name; // eslint-disable-line no-param-reassign
+    organizationFields.title = organizationFields.name.replace(/\W/g, ''); // eslint-disable-line no-param-reassign
+    apiClient.type('organizations').get(this.props.organization.id)
```

Comment:
hmm, not sure how much of an issue it could be with more than one user happening to modify at the same time, but I think I'd like to stay consistent with using the cached resource using the js client for now.